lib/manager/components/ProjectsList.js

```diff
@@ -72,7 +72,8 @@ export default class ProjectsList extends Component<Props> {
     const visibleProjects = projects
       .filter((project) => {
         if (project.isCreating) return true // projects actively being created are always visible
-        return project.name.toLowerCase().indexOf((visibilitySearchText || '').toLowerCase()) !== -1
+        // FIXME why are some project names showing up as null?
+        return !project.name || project.name.toLowerCase().indexOf((visibilitySearchText || '').toLowerCase()) !== -1
```

Comment:
Any chance we can resolve this FIXME?

Reply:
That might need to be a datatools-server fix to make sure projects don't get saved with null names.

Reply:
ibi-group/datatools-server#228